hydroflow_lang/src/graph/ops/join_fused_rhs.rs

```diff
@@ -0,0 +1,68 @@
+use quote::{quote_spanned, ToTokens};
+use syn::parse_quote;
+
+use super::{
+    DelayType, FlowProperties, FlowPropertyVal, OperatorCategory, OperatorConstraints,
+    OperatorWriteOutput, WriteContextArgs, RANGE_0, RANGE_1,
+};
+use crate::graph::PortIndexValue;
+
+/// See `join_fused_lhs`
+///
+/// This operator is identical to `join_fused_lhs` except that it is the right hand side that is fused instead of the left hand side.
```

Comment:
So why is the code so much shorter? (Apologies for not reading detail there... want to get quick feedback to you)

Reply:
The code is much shorter because the rhs version of this operator just calls the lhs version except it swaps the two inputs and then it maps the output to swap (k, (v1, v2)) -> (k, (v2, v1))

This way you don't have to write the whole thing again for the rhs version.

The only reason we need to do it this way is because we don't have a way to configure the stratum-delay function based on arguments passed to the operator. If we ahd that, then we can remove all these different versions of join_fused() (since they all just have different permutations of lhs/rhs blocking)